app/controllers/api/authentications_controller.rb

```diff
@@ -1,4 +1,11 @@
 module Api
   class AuthenticationsController < BaseController
+    def delete_resource(type, id, _data = {})
+      auth = resource_search(id, type, collection_class(:authentications))
+      task_id = auth.delete_in_provider_queue
```

Comment:
should we support authentications subcollection delete ?

i.e. DELETE /api/configuration_script_payloads/:id/authentications/:a_id  (and bulk equiv)